string robot_name
string team
float32 health
bool is_alive
string weapon_type
bool is_planting
bool is_defusing
